src/test/resources/com/puppycrawl/tools/checkstyle/api/fullident/InputFullIdentLiteralNewCondition.java

```diff
@@ -0,0 +1,22 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.coding.UnusedLocalVariableCheck
+allowUnnamedVariables = false
+
+*/
+
+package com.puppycrawl.tools.checkstyle.api.fullident;
+
+public class InputFullIdentLiteralNewCondition {
+    {
+        int j = 20; // violation, 'Unused local variable'
+        nested obj = new nested() {
+            void method() {
+                j += 50;
+            }
+        };
+        obj.getClass();
+    }
+    class nested {
```

Comment:
`Nested` lets keep test code as good formatted if it is not point test.

Reply:
done